lib/cn/kvset.h

```diff
@@ -396,6 +410,14 @@ kvset_get_tree(struct kvset *kvset);
 struct vblock_desc *
 kvset_get_nth_vblock_desc(struct kvset *ks, uint32_t index);
 
+/* MTF_MOCK */
+void
+kvset_set_nodeid(struct kvset *kvset, uint64_t nodeid);
```

Comment:
Does it make sense for kvsets to have a reference to their node somehow whether it a node list entry or a pointer to the node itself?